src/tmx/TmxUtils/src/UdpServer.cpp

```diff
@@ -198,4 +198,22 @@ namespace tmx::utils {
         return -1;
     }
 
+    std::string UdpServer::stringTimedReceive(int maxWait_ms) {
```

Comment:
timedReceive already exists. This method is different because instead of taking in a buffer and populating it, this method assumes the desired output is string, creates a buffer and reads it for you. I will add a comment for the method.